### DISCUSSION TITLE
Why does the data get fetched 10 times?

### DISCUSSION THREAD

**Discussion by jensbroecher** [grid-js/gridjs#1359] [Q&A]
I have this problem that the data is fetched 10 times from the server, and i could not find out why. This is my code:

    function view_order_grid() {

        $('admin_content').html(`
            <div style="max-width: 60px; float: right;" class="swal2-confirm swal2-styled" onclick="download_table_as_csv();">CSV</div>
            <div id="grid_container"></div>
        `);

        const grid = new gridjs.Grid({
            columns: [
                'ID',
                'Status',
                'Date',
                'Name',
                'Last Name',
                'Orders',
                'Address',
                'Cart',
                'Count',
                'Total',
                'Payment'
            ],
            server: {
                url: 'https://afrisupermarket.com/asm_api/?task=admin_get_orders_grid&asm_api_auth=' + asm_api_auth + '',
                then: data => data.map(card => [
                    card.id,
                    card.status,
                    card.tstamp,
                    card.first_name,
                    card.last_name,
                    card.number_of_orders,
                    card.address_user_entry,
                    card.shopping_cart,
                    card.shopping_cart_count,
                    card.shopping_cart_total,
                    card.payment_method
                ]),
                handle: (res) => {
                    // no matching records found
                    if (res.status === 404) return {
                        data: []
                    };

                    if (res.ok) return res.json();

                    throw Error('oh no :(');
                },
            },
            search: {
                enabled: true
            },
            sort: true,
        }).render(document.getElementById("grid_container"));

        // grid.on('rowClick', (...args) => console.log('row: ' + JSON.stringify(args), args));
        // grid.on('cellClick', (...args) => console.log('cell: ' + JSON.stringify(args), args));

        grid.on('rowClick', (...args) => get_order(args[1]._cells[0].data));
    }

<img width="857" alt="Screenshot 2023-06-09 at 14 58 00" src="https://github.com/grid-js/gridjs/assets/6793646/e06f2e8b-a674-4dc4-89b7-3425b7ff29bd">

**Comment by garidium** [comment]
It's a reported bug.....

https://github.com/grid-js/gridjs/issues/1317

Would appreciate it if someone knowledgable would fix it.....
